Tools/docker_run.sh

```diff
@@ -61,6 +61,9 @@ docker run -it --rm -w "${SRC_DIR}" \
 	--env=PX4_UBSAN \
 	--env=TRAVIS_BRANCH \
 	--env=TRAVIS_BUILD_ID \
+	--env=DISPLAY \
+	--volume /tmp/.X11-unix:/tmp/.X11-unix \
+	--volume $HOME/.Xauthority:/home/root/.Xauthority \
```

Comment:
Why `/home/root/.Xauthority`? 

For the typical PX4 containers I believe it's /home/user, but I'm not sure if there's a better way to do this more generally.
```suggestion
	--volume $HOME/.Xauthority:/home/user/.Xauthority \
```

Reply:
You're totally right - using `user` is enough and also works perfectly. Thanks for the hint!